README.rst

```diff
@@ -0,0 +1,143 @@
+universal\_notifications
+========================
+|travis|_ |pypi|_ |codecov|_
+
+**High-level framework for notifications**
+
+This project is intended to provide a convenient way to send notifications using multiple
+notification backends (e.g., e-mail, SMS, push).
+
+--------------
+
+Setting up
+----------
+
+To start using **universal\_notifications** please add ``universal_notifications`` to
+``INSTALLED_APPS`` in your Django project, and then migrate the app:
+``./manage.py migrate universal_notifications``.
+
+Basic usage
+-----------
+
+-  `WebSocket notifications`_
+-  `E-mail notifications`_
+-  `SMS notifications`_
+-  `Push notifications`_
+
+WebSocket notifications
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To have Universal Notifications receive WS notifications (ie. to mark notification as received)
+add to your settings.py:
+
+::
+
+    WS4REDIS_SUBSCRIBER = 'universal_notifications.backend.websockets.RedisSubscriber'
+
+Simple example of using WS notifications:
+
+.. code:: python
+
+    class OrderShippedWS(WSNotification):
+        message = 'order_shipped'
+        serializer_class = OrderSerializer
+
+    # ... somewhere in a view
+    OrderShippedWS(item=order, receivers=[user], context={}).send()
+
+E-mail notifications
+~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    class OrderShippedEmail(EmailNotification):
+        email_name = 'order_shipped'
+        email_subject = _('{{user.full_name}}, Your order no. {{item.pk}} has been shipped.')
+
+    # ... somewhere in a view
+    OrderShippedEmail(item=order, receivers=[user], context={'user': user}).send()
+
+E-mail notifications allow filtering e-mail addreses which are unsubscribed from receiving e-mails
+from your service. This can be achievied by setting the
+``UNIVERSAL_NOTIFICATIONS_UNSUBSCRIBED_MODEL`` property in your project’s settings.
+
+Sample **UnsubscribedModel** should at least contain the **email** property. As result, all
+addresses existing in **UnsubscribedModel** objects will not receive any e-mails.
+
+SMS notifications
+~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    class OrderShippedSMS(SMSNotification):
+        message = _('{{user.full_name}}, Your order no. {{item.pk}} has been shipped.')
+
+        def prepare_receivers(self):
+            return {x.shipping_address.phone for x in self.receivers}
+
+    # ... somewhere in a view
+    OrderShippedSMS(item=order, receivers=[user], context={'user': user}).send()
+
+This feature requires setting the ``UNIVERSAL_NOTIFICATIONS_SMS_FUNC`` property in project’s settings.
```

Comment:
It is not valid.
By default it will use Twilio and here you can override.
Also we have much more settings now so in general we need to update it.